index.html

```diff
@@ -11,12 +11,9 @@
 	</head>
 	
 	<body>
-		
 		<nav>
-
 			<div class="navSpacer">
-				<div>
-					
+				<div>	
 				</div>
```

Comment:
I'm guessing this empty div is part of your spacing - is it necessary? Can the styles (padding/margin) not be applied directly to the 'navSpacer' container?